### PR TITLE
DNS protocol

### PR DESCRIPTION
**Pull request by earlgreyz**
DNS loadtesting executor with a complete `TUTORIAL.md` to match already implemented protocols _(thrift/http)_

### CODE REVIEW

**Comment by earlgreyz** [comment]
Build failing because of https://github.com/pinterest/bender/pull/14 not merged

**Comment by insomniacslk** [comment]
that PR has been stuck for a year, shall we create a new PR with the same fix + tests?

**Comment by jparise** [comment]
@earlgreyz I just merged #14.

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 language: go
 go:
-    - 1.7
+    - 1.9
```
Comment (jparise) [inline .travis.yml]: Would you mind submitting this version bump separately to keep the commit log clean?

**Comment by earlgreyz** [comment]
@jparise done https://github.com/pinterest/bender/pull/22

**Comment by earlgreyz** [comment]
@jparise I've updated the README mentioning the protocol, the DNS library and linking the `TUTORIAL.md`

**Comment by jparise** [comment]
Thanks for the contribution, @earlgreyz!

**Pull request merged**